nmdc_server/ingest/study.py

```diff
@@ -65,27 +65,18 @@ def load(db: Session, cursor: Cursor):
             obj["principal_investigator_id"] = get_or_create_pi(db, pi_name, pi_url, pi_orcid)
             obj["principal_investigator_websites"] = obj.pop("websites", [])
         obj["image"] = get_study_image_data(obj.pop("study_image", []))
+        dois=obj.pop("associated_dois", None)
+        if dois:
+            for doi in dois:
+                doi["doi_value"] = transform_doi(doi.pop("doi_value"))
 
-        publication_dois = [transform_doi(d) for d in obj.pop("publications", [])] + [
-            transform_doi(d) for d in obj.pop("publication_dois", [])
-        ]
-        award_dois = [transform_doi(doi) for doi in obj.pop("award_dois", [])] + [
-            transform_doi(d) for d in obj.pop("emsl_project_dois", [])
-        ]
-        dataset_dois = [transform_doi(doi) for doi in obj.pop("dataset_dois", [])]
+            for doi in dois:
+                upsert_doi(db, doi["doi_value"], doi["doi_category"])
```

Comment:
```suggestion
                upsert_doi(db, doi["doi_value"], DOIType(doi["doi_category"]))
```

Since `upsert_doi` expects a `DOIType` as the last argument, I'd recommend either changing the signature of that method to `str` or just doing the enum value construction here.